src/test/java/ay2021s1_cs2103_w16_3/finesse/logic/commands/bookmarkcommands/ConvertBookmarkExpenseCommandTest.java

```diff
@@ -14,98 +14,98 @@
+        ConvertBookmarkExpenseCommand convertBookmarkExpenseCommand =
+                new ConvertBookmarkExpenseCommand(INDEX_FIRST, dateOfConvertedExpense);
 
         Model expectedModel = new ModelManager(model.getFinanceTracker(), new UserPrefs());
-        Expense convertedExpense = frequentExpenseToBeConverted.convert(dateOfConvertedExpense);
+        Expense convertedExpense = bookmarkExpenseToBeConverted.convert(dateOfConvertedExpense);
         expectedModel.addExpense(convertedExpense);
 
-        String expectedMessage = String.format(ConvertFrequentExpenseCommand.MESSAGE_CONVERT_FREQUENT_EXPENSE_SUCCESS,
+        String expectedMessage = String.format(ConvertBookmarkExpenseCommand.MESSAGE_CONVERT_BOOKMARK_EXPENSE_SUCCESS,
                 convertedExpense);
 
-        assertCommandSuccess(convertFrequentExpenseCommand, model, expectedMessage, expectedModel, true);
+        assertCommandSuccess(convertBookmarkExpenseCommand, model, expectedMessage, expectedModel, true);
     }
 
     @Test
     public void execute_invalidIndexFilteredList_throwsCommandException() {
-        showFrequentExpenseAtIndex(model, INDEX_FIRST);
+        showBookmarkExpenseAtIndex(model, INDEX_FIRST);
 
         Index outOfBoundIndex = INDEX_SECOND;
         Date dateOfConvertedExpense = new Date(VALID_DATE_SPOTIFY_SUBSCRIPTION);
         // ensures that outOfBoundIndex is still in bounds of frequent expense list
-        assertTrue(outOfBoundIndex.getZeroBased() < model.getFinanceTracker().getFrequentExpenseList().size());
+        assertTrue(outOfBoundIndex.getZeroBased() < model.getFinanceTracker().getBookmarkExpenseList().size());
 
-        ConvertFrequentExpenseCommand convertFrequentExpenseCommand =
-                new ConvertFrequentExpenseCommand(outOfBoundIndex, dateOfConvertedExpense);
+        ConvertBookmarkExpenseCommand convertBookmarkExpenseCommand =
+                new ConvertBookmarkExpenseCommand(outOfBoundIndex, dateOfConvertedExpense);
 
-        assertCommandFailure(convertFrequentExpenseCommand, model, MESSAGE_INVALID_FREQUENT_EXPENSE_DISPLAYED_INDEX);
+        assertCommandFailure(convertBookmarkExpenseCommand, model, MESSAGE_INVALID_BOOKMARK_EXPENSE_DISPLAYED_INDEX);
     }
 
     @Test
     public void equals() {
         Date dateOfConvertedExpense = new Date(VALID_DATE_SPOTIFY_SUBSCRIPTION);
-        ConvertFrequentExpenseCommand convertFirstFrequentExpenseCommand =
-                new ConvertFrequentExpenseCommand(INDEX_FIRST, dateOfConvertedExpense);
-        ConvertFrequentExpenseCommand convertSecondFrequentExpenseCommand =
-                new ConvertFrequentExpenseCommand(INDEX_SECOND, dateOfConvertedExpense);
+        ConvertBookmarkExpenseCommand convertFirstFrequentExpenseCommand =
+                new ConvertBookmarkExpenseCommand(INDEX_FIRST, dateOfConvertedExpense);
+        ConvertBookmarkExpenseCommand convertSecondFrequentExpenseCommand =
```

Comment:
```suggestion
        ConvertBookmarkExpenseCommand convertSecondBookmarkExpenseCommand =
```
Similarly for this one